### DISCUSSION TITLE
Email is received for Forgot Password but not received for new User Creation

### DISCUSSION THREAD

**Discussion by innomanik** [kimai/kimai#4979] [General]
Email is properly configured in .env file and email is received for Forgot Password feature but email is not received when a new User is created by admin, how a new registered user will get notified that he's added, with the URL and the credentials added for the user?

**Comment by kevinpapst** [comment]
There is no such feature.

**Reply by innomanik** [reply]
Thanks for the reply, so we should manually notify the newly added users from outside the system? I thought the system would send them an email that "You've been invited to this system blah blah something", I mean it generally happens in every platform invite that we use.
